.buildkite/filebeat/filebeat-pipeline.yml

```diff
@@ -124,8 +124,10 @@ steps:
 
   - group: "Filebeat Extended Tests"
     key: "filebeat-extended-tests"
+    if: build.env("BUILDKITE_PULL_REQUEST") == "false" || build.env("GITHUB_PR_LABELS") =~ /.*(macOS|arm).*/
```

Comment:
Similar comment to https://github.com/elastic/beats/pull/39507/files#r1601236710

we should remove:

```
build.env("BUILDKITE_PULL_REQUEST") == "false" ||
```

so that it doesn't run on merge commits.

Reply:
not an issue after all, we will keep it to mirror what we have on `main`